pkgs/development/python-modules/doorbirdpy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "doorbirdpy";
+  version = "2.1.0";
+
+  src = fetchPypi {
+    pname = "DoorBirdPy";
+    inherit version;
+    sha256 = "ed0e94953cdf96111c7f73c5fcf358f65dc0ff5e47f63fc057bf18ca7512e606";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  # no tests on PyPI, no tags on GitLab
+  doCheck = false;
+
+  pythonImportsCheck = [ "doorbirdpy" ];
+
+  meta = with lib; {
+    description = "Python wrapper for the DoorBird LAN API v0.21";
```

Comment:
I think that it doesn't make sense to add the API version to the description.

The [documentation](https://www.doorbird.com/downloads/api_lan.pdf) says "Revision: 0.29".

Reply:
The README and setup.py say 0.21, but I'll remove it.